x-pack/plugins/case/server/routes/api/cases/configure/get_configure.ts

```diff
@@ -21,11 +22,15 @@ export function initGetCaseConfigure({ caseConfigureService, router }: RouteDeps
 
         const myCaseConfigure = await caseConfigureService.find({ client });
 
+        const { connector, ...caseConfigureWithoutConnector } =
+          myCaseConfigure.saved_objects[0]?.attributes ?? {};
```

Comment:
I am worrying about not defaulting `connector` like that
```
const { connector, ...caseConfigureWithoutConnector } = myCaseConfigure.saved_objects[0]
          ?.attributes ?? { connector: null };
```

Reply:
or `const { connector = null, ...caseConfigureWithoutConnector } =
          myCaseConfigure.saved_objects[0]?.attributes ?? {};`

Reply:
yes but that upset, ts :(, I tried it first